### PR TITLE
Building size

### PR DESCRIPTION
**Pull request by j3r3m1**
Some of the indicators used for building size properties are calculated. I could not add the "building_likelihood_large_building" indicator since it needs the "building_nb_neighboor" indicator value as input.

### CODE REVIEW

File: geoindicators/src/test/groovy/org/orbisgis/BuildingIndicatorsTests.groovy [hunk 1]
```diff
@@ -26,4 +26,27 @@ class BuildingIndicatorsTests {
         }
     }
 
+    @Test
+    void buildingSizeProperties() {
+        def h2GIS = H2GIS.open([databaseName: './target/buildingdb'])
+        String sqlFilePath = System.getProperty("user.dir")+"/src/test/resources/org/orbisgis/data_for_tests.sql"
```
Comment (ebocher) [inline geoindicators/src/test/groovy/org/orbisgis/BuildingIndicatorsTests.groovy]: Why not use getClass().getResourceAsStream

Comment (j3r3m1) [reply geoindicators/src/test/groovy/org/orbisgis/BuildingIndicatorsTests.groovy]: I would love but I do not know how to use it. I have tried 
`this.getClass().getResourceAsStream("../data_for_tests.sql")`
but it does now work very well...

Comment (ebocher) [reply geoindicators/src/test/groovy/org/orbisgis/BuildingIndicatorsTests.groovy]: @SPalominos

Comment (SPalominos) [reply geoindicators/src/test/groovy/org/orbisgis/BuildingIndicatorsTests.groovy]: You can use 
 ``` groovy
String sqlString = new File(this.class.getResource("data_for_tests.sql").toURI()).text
```

**Comment by ebocher** [comment]
Excellent !

**Pull request merged**